runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -446,6 +481,13 @@ public synchronized CorfuRuntime connect() {
                 throw new RuntimeException(e);
             }
         }
+
+        if (loadSmrMapsAtConnect) {
+            FastSmrMapsLoader fastLoader = new FastSmrMapsLoader(this)
+                    .setBatchReadSize(bulkReadSizeForFastLoader)
+                    .setTimeoutInMinutesForLoading(timeoutInMinutesForFastLoading);
+            fastLoader.loadMaps();
+        }
```

Comment:
I am not sure I understand why this is bundled with `connect()` and not exposed as a separate API. Right now we cannot differentiate between network related issues (TCP timeouts for example) and slow `loadSmrMapsAtConnect()`.